assets/controllers/route-controller-manager/route-controller-manager-authentication-reader-rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: kube-system
+  name: system:openshift:authentication-reader-route-controller-manager
+roleRef:
+  kind: Role
+  name: extension-apiserver-authentication-reader
```

Comment:
Where does this role come from? It looks like it's built into some of the kube code?

Reply:
yes, it comes from kube: https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/namespace_policy.go#L77